build.sbt

```diff
@@ -40,34 +40,24 @@ lazy val commonSettings = mimaDefaultSettings ++ Seq(
 )
 
 // needs to be kept in sync with travis-ci
-val PlayVersion = playVersion(sys.env.getOrElse("PLAY_VERSION", "2.8.0-M3"))
+val PlayVersion = playVersion(sys.env.getOrElse("PLAY_VERSION", "2.8.0-RC1"))
 
 // Version used to check binary compatibility
-val mimaPreviousArtifactsVersion = "7.0.0"
-
-def mimaPreviousVersionExcludeScala213(scalaVersion: String, modules: Set[ModuleID]): Set[ModuleID] = {
-  CrossVersion.partialVersion(scalaVersion) match {
-    case Some((2, v)) if v >= 13 => Set.empty
-    case _                       => modules
-  }
-}
+val mimaPreviousArtifactsVersion = "7.0.1"
```

Comment:
I use `7.0.1` here because that was the first release that published Scala 2.13 artifacts. From `7.0.0` to `7.0.1` [there was no breaking change](https://github.com/playframework/play-mailer/compare/7.0.0...7.0.1) anyway.
However, this will set to `8.0.0` as soon it's released anyway, so that config should not really matter for now.